js/everything.js

```diff
@@ -378,20 +378,21 @@ function toggleElement(event, toggleID, buttonText) {
  */
 function getCurrentVideoID() {
     var v = URI(window.location).search(true).v;
-    if (!v) {
-        return null;
-    }
-
-    if (Array.isArray(v)) {
-        v = v[0];
+    var r;
```

Comment:
Since only 1 possible if/else can be executed, we can just update the `v` variable - we don't need `r` at all.